source/api_documentation/calls_in_progress_api/_get_calls_by_phone_number.rst

```diff
@@ -19,22 +22,47 @@
     {
       "current_calls": [
         {
-          "transaction_id": "01FMTK644Z9D-3FPEY1REX8JSZB",
+          "transaction_id": "ABCD1234-ABCD1234",
           "call_status": "bridged",
           "external_unique_id": null,
           "calling_phone_number": "+15309999999",
           "destination_phone_number": "+18555595599",
-          "call_start_time_utc": "2022-12-10T13:45:00.000Z",
-          "transfer_start_time_utc": "2022-12-10T13:49:00.000Z",
-          "custom_data": {}
+          "called_phone_number": "+18557174046",
+          "bridge_start_time": "2023-04-03T16:02:36-07:00",
+          "phone_type": "Mobile",
+          "custom_data": {
+                "utm_source": {
+                    "value": "google.com",
+                    "source": "VirtualLine"
+                },
+                "demographics_data": {
+                    "display_name_data_append": "Stan Bobman",
```

Comment:
Is this real/valid demographic data from a live request?

Reply:
It is, although it is from my number (which was changed for this example), so I had thought it was fake. I'll edit it a bit just in case that is a real person who used to have my phone number.